lib/config/options/index.ts

```diff
@@ -2604,11 +2604,11 @@ const options: RenovateOptions[] = [
     default: {
```

Comment:
I'm not sure which Conventional Commit scope to use for the PR title. My changes are user-facing, maybe it needs to be `fix` or `feat` instead of `refactor`?

Reply:
fix should be ok